### PR TITLE
feat(route): Add route to SISU news.

### PR DESCRIPTION
**Pull request by Duuckjing**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/shisu/news
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [v2 Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [v2 路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Documentation / 文档说明
- [x] Full text / 全文获取
  - [x] Use cache / 使用缓存
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by Duuckjing** [comment]
It seems that the feed is still version 1.0.

File: lib/v2/shisu/maintainer.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    news: ['duuckjing'],
```
Comment (TonyRL) [inline lib/v2/shisu/maintainer.js:2]: https://github.com/DIYgod/RSSHub/blob/bf1025e0e3243b870f9909780251b12dd3cbb00c/lib/v2/shisu/router.js#L2
```suggestion
    '/news': ['Duuckjing'],
```

File: lib/v2/shisu/news.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+const url = 'https://news.shisu.edu.cn';
+
+module.exports = async (ctx) => {
+    const { data: res } = await got(String(url));
```
Comment (TonyRL) [inline lib/v2/shisu/news.js:9]: `url` is already of type `String`. There's no need to recast it to the same type.
docs: https://docs.rsshub.app/joinus/advanced/script-standard#casting
https://github.com/DIYgod/RSSHub/blob/bf1025e0e3243b870f9909780251b12dd3cbb00c/lib/v2/shisu/news.js#L6
```suggestion
    const { data: res } = await got(url);
```

File: lib/v2/shisu/news.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        .map((i0) => {
+            const i = $(i0);
+            return {
+                title: i.find('a').text(),
+                link: `${url}${i.find('a').attr('href')}`,
+                pubDate: timezone(parseDate(i.find('.in-con02 > span:nth-child(2)').text(), 'YYYY-MM-DD'), +8),
+                description: i.find('.default-txt').text(),
+                language: 'zh-cn',
+                image: i.find('img').attr('src'),
+                icon: 'https://www.shisu.edu.cn/assets/images/sisu.ico',
```
Comment (TonyRL) [inline lib/v2/shisu/news.js:22]: `item` in an RSS feed does not contain these fields.
```suggestion
```
docs: https://docs.rsshub.app/joinus/advanced/advanced-feed

File: lib/v2/shisu/radar.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'shisu.edu.cn': {
+        _name: '上海外国语大学',
+        news: [
+            {
+                title: '上外新闻',
+                docs: 'https://docs.rsshub.app/routes/university#shang-hai-wai-guo-yu-da-xue',
+                source: ['/*'],
+                target: '/shisu/news/:category?',
```
Comment (TonyRL) [inline lib/v2/shisu/radar.js:9]:
```suggestion
                source: ['/:category/index.html'],
                target: '/shisu/news/:category',
```

**Pull request closed without merging**